.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: android
+
+jdk:
+ - oraclejdk7
+ - oraclejdk8
+
+matrix:
+  allow_failures:
+    - jdk: oraclejdk7
```

Comment:
Yes, but you have to do it in the build.gradle file as a jvmOpt to `test {}` (or something like that). It isn't really possible to do it from the command line just from Travis, unless you are wanting to configure how much memory gradle uses for building in which case you would add it to the `GRADLE_OPTS` environment variable.